packages/opds-web-client/src/stylesheets/custom-themes/_theme-demo.scss

```diff
@@ -0,0 +1,6 @@
+//
+// Demo theme colors
+//
+
+$blue: #3a7dfd;
+$darkblue: #234c99;
```

Comment:
This isn't used, right? It won't do anything unless you override some of the other vars..